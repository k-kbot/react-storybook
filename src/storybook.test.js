import path from 'path';
import initStoryshots from '@storybook/addon-storyshots';
import { imageSnapshot } from '@storybook/addon-storyshots-puppeteer';

const getMatchOptions = ({ context: { fileName } }) => {
  const snapshotPath = path.join('src/__imageSnapShots__');
  return { customSnapshotsDir: snapshotPath };
};

initStoryshots({
  test: imageSnapshot({
    getMatchOptions,
  }),
});
