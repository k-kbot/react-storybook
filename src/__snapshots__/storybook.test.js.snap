// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/Board Default 1`] = `
<div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
  </div>
</div>
`;

exports[`Storyshots Components/Board Filled With Delta 1`] = `
<div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
  </div>
</div>
`;

exports[`Storyshots Components/Board Filled With O 1`] = `
<div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      O
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      O
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      O
    </button>
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      O
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      O
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      O
    </button>
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      O
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      O
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      O
    </button>
  </div>
</div>
`;

exports[`Storyshots Components/Board Filled With X 1`] = `
<div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      X
    </button>
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      X
    </button>
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      X
    </button>
  </div>
</div>
`;

exports[`Storyshots Components/Game Default 1`] = `
<div
  className="game"
>
  <div
    className="game-board"
  >
    <div>
      <div
        className="board-row"
      >
        <button
          className="square"
          onClick={[Function]}
        />
        <button
          className="square"
          onClick={[Function]}
        />
        <button
          className="square"
          onClick={[Function]}
        />
      </div>
      <div
        className="board-row"
      >
        <button
          className="square"
          onClick={[Function]}
        />
        <button
          className="square"
          onClick={[Function]}
        />
        <button
          className="square"
          onClick={[Function]}
        />
      </div>
      <div
        className="board-row"
      >
        <button
          className="square"
          onClick={[Function]}
        />
        <button
          className="square"
          onClick={[Function]}
        />
        <button
          className="square"
          onClick={[Function]}
        />
      </div>
    </div>
  </div>
  <div
    className="game-info"
  >
    <div>
      Next player: X
    </div>
    <ol>
      <li>
        <button
          onClick={[Function]}
        >
          Go to game start
        </button>
      </li>
    </ol>
  </div>
</div>
`;

exports[`Storyshots Components/Square Default 1`] = `
<button
  className="square"
  onClick={[Function]}
/>
`;

exports[`Storyshots Components/Square Filled With X 1`] = `
<button
  className="square"
  onClick={[Function]}
>
  X
</button>
`;
